spec/lib/reporting/fraud_metrics_lg99_report_spec.rb

```diff
@@ -118,7 +118,7 @@
   describe '#user_days_to_reinstatement_avg' do
     context 'where there are reinstated users' do
       it 'returns average time to reinstatement' do
-        expect(report.user_days_to_reinstatement_avg).to eq(3.0)
+        expect(report.user_days_to_reinstatement_avg).to be_within(0.1).of(3.0)
```

Comment:
could we make the test more precise if we froze time + used hardcoded times for the `suspended_at`, `reinstated_at`?

Reply:
or round it in the test?
```suggestion
        expect(report.user_days_to_reinstatement_avg.round).to eq(3)
```

Reply:
I think this is more about the time of day when a user gets created in the test db vs any frozen time after the fact. We can certainly round in the test, it just won't be as realistic.

Reply:
right -- we have control over when the user gets created, I think we should just set that, especially for something that's deterministic

Reply:
I could create the user with `created_at: ` from a let and then do `reinstated: created_at + 3.days` ...

Reply:
yeah let's do that!

Reply:
I've updated the screenshots to show expected user attributes during the tests